pipeline/terraform/env.sh.example

```diff
@@ -35,4 +35,37 @@ export TF_VAR_ecs_asg_min_size=2
 export TF_VAR_ecs_asg_max_size=2
 
 # HTTPS certificate ARN from AWS certificate manager
-export TF_VAR_certificate_arn=arn:aws:acm:us-east-1:123412341234:certificate/12341234-3144-4c26-ba3c-123412341234
\ No newline at end of file
+export TF_VAR_certificate_arn=arn:aws:acm:us-east-1:123412341234:certificate/12341234-3144-4c26-ba3c-123412341234
+
+#schema name for the rds-mysql database
+export TF_VAR_identifier=demodb
+
+#database engine specifying "mysql" as the database server of choice in RDS
+export TF_VAR_engine=mysql
+
+#Version of mysql server to launch in RDS
+export TF_VAR_engine_version=5.7.19
+
+# db instance class to launch in RDS
+export TF_VAR_instance_class=db.t2.large
+
+# if you'd like the database encrypted
+export TF_VAR_storage_encrypted=false
+
+# this is the allocated storage in GB
+export TF_VAR_allocated_storage=5
+
+# this is the maintenance window during which maintenance (i.e. upgrade) will occur.
+export TF_VAR_maintenance_window=Mon:00:00-Mon:03:00
+
+# this is the window during which daily backup will occur
+export TF_VAR_backup_window=03:00-06:00
+
+# this is the superuser created for database admin purposes
+export TF_VAR_username=user
+
+# this is the password of the db superuser
+export TF_VAR_password=YourPwdShouldBeLongAndSecure!
```

Comment:
Can we make this configurable or do you override this during execution of the script?

Reply:
This is just the sample file, which you copy (to `env.sh`) and edit with real values.

Reply:
Yup, understood.  Maybe we need a header comment at the top of the file to explain that just a little bit further, even though the filename has "example" in it.

Reply:
Added this comment at the top of the file 
`# THIS IS AN EXAMPLE FILE. Change the values of each variable according to your specification`

Reply:
Cool!